pkg/controller/replication_controller_test.go

```diff
@@ -374,4 +375,5 @@ func TestWatchControllers(t *testing.T) {
 	case <-time.After(100 * time.Millisecond):
 		t.Errorf("Expected 1 call but got 0")
 	}
+	close(stopCh)
```

Comment:
nit: ```defer close(stopCh)``` up at 366 is slightly better... it shuts down the controller even if something panics in this function.